src/core/Form/SearchInput/__snapshots__/SearchInput.test.tsx.snap

```diff
@@ -276,7 +295,7 @@ exports[`calling render with the same component on the same container does not r
       class="c4 fi-search-input_input-container fi-text-input_container"
     >
       <input
-        aria-describedby="test-id-statusText"
+        aria-describedby=""
```

Comment:
is this due to test setup or something that occurs with real use cases? Could empty aria-describedby have undesired side-effects?

Reply:
Good find.

If I read correctly, previously the generated id for `statusText` was always added to `aria-describedby` even if the were no `statusText` given. Now, after changes the generated id's (_statusText and the new hintText_) are only added when they are given.

Empty `aria-describedby` might cause small pause when screenreader reads it according to this discussion:
https://developer.paciellogroup.com/blog/2018/09/describing-aria-describedby/#div-comment-4779

Reply:
I came across that discussion earlier too. How I understood it was that the delay was caused by the element referred to by the `aria-describedby` being empty. Still always better to have a clean solution :+1:

Reply:
Seems to be clean now, no attribute at all if no related descriptions are given.